app/assets/javascripts/backbone/views/breakeven_product_view.coffee

```diff
@@ -0,0 +1,83 @@
+class MS.Views.BreakevenProductView extends Backbone.View
+
+  # The view is called from the editiable table view.
+
+  events:
+    'change input.price': 'changed'
+    'change input.cost': 'changed'
+    'change input.percentage_of_sales': 'changed'
+
+  initialize: (options) =>
+    @total_fixed_costs = @readFromDom('total_fixed_costs')
+    @Q = 0.0
+
+    # Refactor to separate all calculations from Dom
+    @_cost = @readFromDom('cost')
+    @_price = @readFromDom('price')
+    @_percentageOfSales = @readPercentageOfSalesFromDom()
+
+    @updateDom()
+
+  updateDom: ->
+    @writeToDom('revenue', @revenue())
+    @writeToDom('quantity', @quantity())
+    @writeToDom('quantity_display_value', @quantity())
+    @writeToDom('total_cost', @totalCost())
+
+  isValid: =>
```

Comment:
You shouldn't need `=>` on these methods. Note e.g. that `updateDom` doesn't have it but `@` stilll works.